splunk_add_on_ucc_framework/templates/input.helper-init-template

```diff
@@ -0,0 +1,87 @@
+import json
+import logging
+
+import import_declare_test
+from solnlib import conf_manager, log
+from splunklib import modularinput as smi
+
+
+ADDON_NAME = "{{addon_name}}"
+
+
+def logger_for_input(input_name: str) -> logging.Logger:
+    return log.Logs().get_logger(f"{ADDON_NAME.lower()}_{input_name}")
+
+
+def get_account_api_key(session_key: str, account_name: str):
+    cfm = conf_manager.ConfManager(
+        session_key,
+        ADDON_NAME,
+        realm=f"__REST_CREDENTIAL__#{ADDON_NAME}#configs/conf-{{addon_rest_root}}_account",
+    )
+    account_conf_file = cfm.get_conf("{{addon_name}}_account")
+    return account_conf_file.get(account_name).get("api_key")
+
+
+def get_data_from_api(logger: logging.Logger, api_key: str):
+    logger.info("Getting data from an external API")
+    dummy_data = [
+        {
+            "line1": "hello",
+        },
+        {
+            "line2": "world",
+        },
+    ]
+    return dummy_data
+
+
+def validate_input(definition: smi.ValidationDefinition):
+    return
+
+
+def stream_events(inputs: smi.InputDefinition, event_writer: smi.EventWriter):
+    # inputs.inputs is a Python dictionary object like:
```

Comment:
Can we remove this commented code?

Reply:
I think it's okay to leave it there. It might give people a useful hint.